### PR TITLE
Update ConvertDataOperation.md

### PR DESCRIPTION
**Pull request by ginalee-dotcom**
## Description
Update with JSON to FHIR information.

## Related issues
N/A

## Testing
N/A

## FHIR Team Checklist
- [ ] **Update the title** of the PR to be succinct and less than 50 characters
- [ ] **Add a milestone** to the PR for the sprint that it is merged (i.e. add S47)
- [ ] Tag the PR with the type of update: **Bug**, **Dependencies**, **Enhancement**, or **New-Feature**
- [ ] Tag the PR with Azure API for FHIR if this will release to the Azure API for FHIR managed service (CosmosDB or common code related to service)
- [ ] Tag the PR with Azure Healthcare APIs if this will release to the Azure Healthcare APIs managed service (Sql server or common code related to service)
- Review [squash-merge requirements](https://github.com/microsoft/fhir-server/blob/master/SquashMergeRequirements.md)

### Semver Change ([docs](https://github.com/microsoft/fhir-server/blob/master/docs/Versioning.md))
Patch|Skip|Feature|Breaking (reason)

### CODE REVIEW

File: docs/ConvertDataOperation.md [hunk 1]
```diff
@@ -23,11 +23,13 @@ Make a call to ```<<FHIR service base URL>>/$convert-data``` with the [Parameter
 | Parameter Name      | Description | Accepted values |
 | ----------- | ----------- | ----------- |
 | inputData      | Data to be converted. | For `Hl7v2`: string <br> For `Ccda`: XML|
```
Comment (LTA-Thinking) [inline docs/ConvertDataOperation.md:25]: Should JSON have a value here?

Comment (ginalee-dotcom) [reply docs/ConvertDataOperation.md]: Fixed it :) Thanks RB!

**Review by LTA-Thinking — APPROVED** [review]

**Comment by ginalee-dotcom** [comment]
@LTA-Thinking @namadabu Thank you for reviewing this! May I know when this can be merged? Also adding @CaitlinV39 for reference

**Pull request merged**